### ISSUE TITLE
Графики работ.

### ISSUE THREAD

**Issue by Linda4ka** [Contabilizare/c5#46]
Нет функции автоматического заполнения календаря в документе Графики на основании введенных в справочник Праздники праздничных дней.
Вручную играться очень неудобно. Рассмотрите пожалуйста вариант автоматического заполнения праздничных дней. Или если он есть и я его не увидела, подскажите где смотреть.Спасибо.
P.S. На этом фоне многие клиенты просят чтобы автоматически заполнялись и предпраздничные дни, с графиком работы 7 часов для 8-часового рабочего дня.

**Comment by Linda4ka** [comment]
Даже не так! Календарь некорректно заполняется даже при заполненном справочнике Праздники. Все равно проставляются 8, а должно быть пусто! Так будет вернее поставлен вопрос!!!

**Comment by Nullarity** [comment]
Да, вы совершенно правы с т.з. графика как основы для начисления ЗП. Но в к5 сделано чуть иначе. Графики работ и праздники теперь не завязаны друг на друге и могут заполняться раздельно. В графике задается только норма рабочего времени, а непосредственно определение кол-ва отработанных дней/часов выполняется в момент начисления зарплаты.
Так сделано по ряду причин:
- Не нужно помнить последовательность ввода Праздники / потом Графики
- Если графиков много, не нужно их перезаполнять в случае сдвига или объединения праздников

То есть хотелось достичь такого, чтобы пользователь в "два клика" создавал нужные ему графики, вел праздники по мере поступления информации, а все остальное делала программа.

Но вот с предпраздничными днями у нас в программе упущение, сейчас автоматом -1 час не делается, записал задачку. Спасибо!

**Comment by Nullarity** [comment]
@Linda4ka я добавил #65, дополняйте если что
